src/rail/cli/rail_pipe/pipe_scripts.py

```diff
@@ -16,6 +16,24 @@
 from rail.cli.rail_pipe.pipe_options import RunMode
 
 
+S3DF_SLURM_OPTIONS: list[str] = [
+    "-p",
+    "milano",
+    "--account",
+    "rubin:commissioning@milano",
+    "--mem",
+    "16448",
+    "--parsable",
+]
+NERSC_SLURM_OPTIONS: list[str] = [
```

Comment:
I think we want the following for perlmutter:
```
"--account",
"m1727",
"--constraint",
"cpu",
"--qos",
"regular"
```
alongside any options related to memory, cpu usage, etc.

Reply:
thanks!

Reply:
though if any of the rail estimators run on GPUs, we may need to specify those as distinct sites...?